### PR TITLE
Finish create commad

### PR DESCRIPTION
**Pull request by Andilun**
Creation of GitHub repository is no longer the default option. GitHub repo is only created if supplied with --github.

A local git repository will be made by default if --github is not supplied.
All files before poetry is installed is added and committed. (poetry.lock is not in initial commit.)

Split build command into functions poetry_install and install_ipykernel.

### CODE REVIEW

File: src/ssb_project_cli/ssb_project/app.py [hunk 1]
```diff
@@ -251,31 +286,38 @@ def create(
...
-        if not skip_github and is_github_repo(github_token, project_name):
+        if add_github and is_github_repo(github_token, project_name):
             raise ValueError(f"The repo {project_name} already exists on GitHub.")
 
-    create_project_from_template(project_name, description)
+    if add_github and description == "":
+        description = request_project_description()
 
+    create_project_from_template(project_name, description)
     # Create empty folder on root
```
Comment (mmwinther) [inline src/ssb_project_cli/ssb_project/app.py:300]: I think these comments could be removed, they were just meant as TODOs during the Hack

File: src/ssb_project_cli/ssb_project/app.py [hunk 2]
```diff
@@ -251,31 +286,38 @@ def create(
...
-        git_repo_dir = DEFAULT_REPO_CREATE_PATH.joinpath(project_name)
         make_git_repo_and_push(github_token, repo_url, git_repo_dir)
 
         print("Set branch protection rules.")
         set_branch_protection_rules(github_token, project_name)
+    else:
+        # Wont commit poetry.lock, poetry install not called yet
+        make_and_init_git_repo(git_repo_dir)
 
     # 4. Add metadata about creation
```
Comment (mmwinther) [inline src/ssb_project_cli/ssb_project/app.py:322]: We can remove all of this. If we really need the code in the future we can get it from the git history

**Review by mmwinther — COMMENTED** [review]
Looks great and works well, nice job 👍 

Just noticed some extraneous comments left over from the Hack, would be good to get rid of them now.

File: src/ssb_project_cli/ssb_project/app.py (hunk 1 above)
Comment (Andilun) [reply src/ssb_project_cli/ssb_project/app.py]: Removed all TODOs from hack

File: src/ssb_project_cli/ssb_project/app.py (hunk 2 above)
Comment (Andilun) [reply src/ssb_project_cli/ssb_project/app.py]: ^

**Review by mmwinther — APPROVED** [review]
Nice job 👍

**Pull request merged**